src/conn/conn_api.c

```diff
@@ -1325,13 +1325,23 @@ __conn_reconfigure(WT_CONNECTION *wt_conn, const char *config)
     WT_SESSION_IMPL *session;
 
     conn = (WT_CONNECTION_IMPL *)wt_conn;
-
+    
     CONNECTION_API_CALL(conn, session, reconfigure, config, cfg);
     ret = __wt_conn_reconfig(session, cfg);
 err:
     API_END_RET(session, ret);
 }
 
+/*
+ * __conn_get_home --
```

Comment:
Didn't pick this one up earlier - sorry! This should be: 

`__conn_get_configuration` instead of `__conn_get_home`

I believe this will fix the `s_all` issue we have in the evergreen patch failure.